test/cases/base_test.rb

```diff
@@ -810,6 +812,14 @@ def test_custom_prefix
     assert_equal [:person_id].to_set, StreetAddress.__send__(:prefix_parameters)
   end
 
+  def test_has_one_finder_uses_site_path
+    ShippingLabel.site = "http://37s.sunrise.i:3000/api/v1"
+    Order.site = "http://37s.sunrise.i:3000/api/v1"
+    Order.send(:has_one, :shipping_label)
```

Comment:
This is changing the global state so I think we should move all those calls to the class definition in the fixtures